gaei_cpp/triangle_direction.hpp

```diff
@@ -0,0 +1,25 @@
+﻿#pragma once
+#include <vector>
+#include <array>
+#include "vertex.hpp"
+#include "vector_utl.hpp"
+
+namespace gaei{
+void triangle_direction_judege(std::vector<gaei::vertex<gaei::vec3f, gaei::color>>& vertexes, std::vector<std::array<int,3>>& as) {
```

Comment:
点群を変更しないならば`const`参照にした方が良い
```suggestion
void triangle_direction_judege(const std::vector<gaei::vertex<gaei::vec3f, gaei::color>>& vertexes, std::vector<std::array<int,3>>& as) {
```